src/Apps/Artwork/Components/ArtworkAuctionCreateAlertHeader/SuggestedArtworksModal.tsx

```diff
@@ -12,20 +11,17 @@ interface SuggestedArtworksModalProps {
 export const SuggestedArtworksModal: FC<SuggestedArtworksModalProps> = ({
   onClose,
 }) => {
-  const { t } = useTranslation()
-  const { criteria } = useSavedSearchAlertContext()
+  const { criteria, entity } = useSavedSearchAlertContext()
+  const artistName = entity.defaultCriteria?.artistIDs?.[0].displayValue ?? ""
 
   return (
     <ModalDialog
       width={["100%", 700]}
       onClose={onClose}
-      title={t(
-        "artworkPage.artworkAuctionCreateAlertHeader.suggestedArtworksModal.title"
-      )}
```

Comment:
How do we decide when to add translations?

Reply:
In the meantime it was recommended not to use translations for now, so in this specific case I removed it only for visibility purposes until there is a decision about translations in general.